src/applications/personalization/profile/tests/fixtures/communication-preferences/get-200-maximal.json

```diff
@@ -6,65 +6,35 @@
       "communicationGroups": [
         {
           "id": 1,
-          "name": "Health Care",
```

Comment:
I updated this existing file with more recent data from `GET https://dev-api.va.gov/v0/profile/communication_preferences`